zcash_client_backend/src/data_api/testing/pool.rs

```diff
@@ -2511,21 +2511,22 @@ where
     // Truncate back to the reorg height, but retain the block cache.
     st.truncate_to_height_retaining_cache(reorg_height);
 
-    // The following error-prone tree state is generated by the current truncate implementation:
+    // The following error-prone tree state is generated by the a previous (buggy) truncate
```

Comment:
5a599f15d21d0f132809ecdbadbf68a663ec82f0:
```suggestion
    // The following error-prone tree state is generated by a previous (buggy) truncate
```